test/Grpc.Net.Client.Tests/Balancer/ConnectionManagerTests.cs

```diff
@@ -115,12 +116,14 @@ public async Task PickAsync_ChannelStateChangesWithWaitForReady_WaitsForCorrectE
             transportFactory.Transports[i].UpdateState(ConnectivityState.TransientFailure);
         }
 
+        await BalancerWaitHelpers.WaitForSubchannelsToBeReadyAsync(logger, clientChannel, 0);
+
         var pickTask2 = clientChannel.PickAsync(
             new PickContext { Request = new HttpRequestMessage() },
             waitForReady: true,
             CancellationToken.None).AsTask().DefaultTimeout();
 
-        Assert.IsFalse(pickTask2.IsCompleted);
+        Assert.IsFalse(pickTask2.IsCompleted, "PickAsync should wait until an subchannel is ready.");
```

Comment:
I don't get it. You wait for subchannels to be ready, assert that PickAsync doesn't complete, and then say PickAsync shouldn't complete until subchannels are ready. (Also, the grammar is wrong in the sentence, I assume you mean "any subchannels are ready" or "a subchannel is ready")

Reply:
Ah wait, now I understand, the `expectedCount` value is `0`, maybe use named params.